src/hotspot/cpu/aarch64/aarch64_vector.ad

```diff
@@ -2854,7 +2854,8 @@ instruct reduce_addL_sve(iRegLNoSp dst, iRegL isrc, vReg vsrc, vRegD tmp) %{
 %}
 
 // reduction addF
-instruct reduce_add2F_neon(vRegF dst, vRegF fsrc, vReg vsrc) %{
+
+instruct reduce_non_strict_order_add2F_neon(vRegF dst, vRegF fsrc, vReg vsrc) %{
```

Comment:
Now that you have changed the name of the method, you should also change the `format` in all of the methods.

Reply:
Oh no! Sorry I missed that. Will do that right away.

Reply:
I have updated the `format` for the rules I changed. Thanks for spotting it.